lib/tasks/update_database_records_in_big_query.rake

```diff
@@ -1,8 +0,0 @@
-namespace :database_records do
-  desc 'Updates Big Query user and approver tables with latest records from DSI'
-  namespace :in_big_query do
-    task update: :environment do
-      ExportDsiUsersToBigQueryJob.perform_later
```

Comment:
Again, I thought this was how Steven got his data?

Reply:
same as before